web/packages/shared/components/Notification/Notification.tsx

```diff
@@ -150,7 +150,7 @@ function getRenderedContent(
           `}
         >
           <Text
-            fontSize={14}
+            fontSize={13}
```

Comment:
This is a shared component now so we shouldn't really make sneaky changes like that, I'd at least put it in a separate PR. This way the rest of the team has a chance to review it and it can also be backported independent of config changes (rather than having different notification font sizes between versions). I do agree that it looks better though.